user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_SAML/SAML_using_Entra_ID.md

```diff
@@ -0,0 +1,474 @@
+
+   - Microsoft Graph > User.Read – Delegated – Sign in and read user profile
+
+   - Microsoft Graph > User.Read.All – Delegated – Read all users' full profiles
+
+1. Open DataMiner Cube and log in with an existing Administrator account.
+
+1. Add the users/groups as described under [Adding a user](xref:Adding_a_user) and [Adding a user group](xref:Adding_a_user_group). Select to add an existing user or group in order to add users and groups available on Entra ID.
+
+1. When you have added the necessary users, configure their permissions. See [Configuring a user group](xref:Configuring_a_user_group).
+
+   Users will now be able to log in to DataMiner with any of the Entra ID user accounts you have added, using either the domain and username (DOMAIN\\user) or the email address.
+
+***
+
+### Configuring automatic creation of users authenticated by Entra ID using SAML
+
+From DataMiner 10.2.0/10.1.12 onwards, users authenticated by Entra ID using SAML can be automatically created and assigned to groups in DataMiner. This is often referred to as JIT (Just-In-Time) Provisioning.
+
+There are two ways to configure this setup: with or without group claims.
+
+- With group claims, when a user logs in for the first time, they will be added to an existing group that exists both in Entra ID and in DataMiner.
+
+- Without group claims, any user that logs in for the first time will be added to one default security group, which you will need to configure during the setup detailed below. This is easier to configure, but has the disadvantage that user information that is created will not be updated.
+
+#### [With group claims](#tab/tabid-3)
+
+1. Navigate to ``portal.azure.com`` and log in.
+
+1. In the search box at the top, enter *Enterprise applications* to go to the Enterprise Applications page.
+
+1. Select the application you created for DataMiner.
+
+1. In the pane on the left on your DataMiner application page, click *Single sign-on*.
+
+1. Next to *Attributes & Claims*, click *Edit*.
+
+   ![Edit attributes & claims](~/user-guide/images/SAML_Edit_claim.png)
+
+1. Click *Add a group claim*.
+
+   ![Add a group claim](~/user-guide/images/SAML_Add_group_claim.png)
+
+1. Configure the group claim:
+
+   1. Select to include *Groups assigned to the application*.
+
+      With this option, the groups you configured when [setting up the application](#setting-up-a-microsoft-entra-id-enterprise-application) will be included in the group claim.
+
+   1. Under *Source attribute*, we recommend selecting *sAMAccountName*.
+
+   1. Optionally, expand the advanced options to add a filter or customize the name of the group claim.
+
+   1. Click *Save*.
+
+   ![Group claim configuration](~/user-guide/images/SAML_Group_claim_config.png)
+
+   > [!NOTE]
+   > The account name of the group will only be sent via SAML when the groups are synchronized. Otherwise, the ID of the group will be sent instead.
```

Comment:
As a clarification here: where the groups need to be synchronized from is an on prem AD, I'm also looking into a new option added recently by microsoft to send the names of "cloud-only groups" (their words) I am trying to test this today to confirm this works as I expect. If so, we could also add that as an alternative option

Reply:
I'd suggest that we add this in a new pull request later; then we can already merge these improvements, as they'll make quite a difference for the users already.

Reply:
I would just add the "synchronized from an on prem AD" to the docs now, since that can be very confusing to read